src/components/Box/BoxCoreFunctions.scss

```diff
@@ -0,0 +1,45 @@
+@use "sass:string";
+@use "sass:list";
+
+// BOX core functions: mock functions
+// These function should be imported from "monday-ui-styles"
```

Comment:
if they should be imported from monday ui style let's remove it before merge the PR?